lib/app/components/pages/dashboard/views/dashboard_view.dart

```diff
@@ -19,11 +17,9 @@ class DashboardView extends GetView<DashboardController> {
             child: Obx(
               () => IndexedStack(
                 index: controller.tabService.tabIndex.value,
-                children: const [
-                  TabRootView(initialRoot: Routes.home),
-                  TabRootView(initialRoot: Routes.chat),
-                  TabRootView(initialRoot: Routes.mypage)
-                ],
+                children: controller.navigationInfo.map((info) =>
+                    TabRootView(initialRoot: info.initialRoot)
+                ).toList(),
```

Comment:
```suggestion
                children: NavigationUtil.tabRootViews,
```
`NavigationUtil`クラスにまとめちゃったのでこのようにしましょう

Reply:
```suggestion
                children: tabRootViews,
```
シンプルに`tabRootViews`だけにしました。